src/app/operations/operations.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { OperationsContainerComponent } from './operations-container.component';
+import { OperationContainerComponent } from './operation-container.component';
+import { OperationDetailsComponent } from './operation-details.component';
+import { OperationSamplesComponent } from './operation-samples.component';
+import { OperationResourceComponent } from './operation-resource.component';
+import { OperationPathComponent } from './operation-path.component';
+import { OperationParametersComponent } from './operation-parameters.component';
+import { OperationSampleRequestComponent } from './operation-sample-request.component';
+import { OperationParameterComponent } from './operation-parameter.component';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  exports: [
+    OperationsContainerComponent
+  ],
+  declarations: [OperationContainerComponent, OperationDetailsComponent, OperationSamplesComponent, OperationsContainerComponent, OperationResourceComponent, OperationPathComponent, OperationParametersComponent, OperationSampleRequestComponent, OperationParameterComponent]
```

Comment:
This is a really long line. Can we limit line length to 100 chars? That's about how many chars you can have on a line without it wrapping in most editors that allow two editor windows side by side.
We can change `max-line-length` in `tslint.json` to `100`.
On a related note - try running `ng lint` - there's a lot of warnings